kraken-build/src/kraken/std/cargo/__init__.py

```diff
@@ -427,6 +427,7 @@ def cargo_publish(
     project: Project | None = None,
     version: str | None = None,
     cargo_toml_file: Path = Path("Cargo.toml"),
+    allow_overwrite: bool = False,
```

Comment:
FYI because this changes semantics of the task, I will release this as a minor update.